scripts/globals/items/bravura.lua

```diff
@@ -0,0 +1,26 @@
+-----------------------------------------
+-- ID: 18294
+-- Bravura
+-- Additional Effect: Evasion Down -30, 40 second Duration
+-----------------------------------------
+
+require("scripts/globals/status");
+require("scripts/globals/magic");
+
+-----------------------------------
+-- onAdditionalEffect Action
+-----------------------------------
+
+function onAdditionalEffect(player,target,damage)
+    local chance = 10;
+
+    if (math.random(0,99) >= chance) then
+        return 0,0,0;
+    else
+        target:delStatusEffect(EFFECT_EVASION_DOWN)
```

Comment:
might be better to check the status effect first before attempting to remove it (get rid of the not and stick this line inside the check, move the adding of the status effect outside of it)